dips/dip-10.md

```diff
@@ -101,130 +110,39 @@ struct DiemIdDomainEvent {
 ```
-
-| Field 	   | Type 	     | Required? 	| Description 	           |
-|-------	   |------	     |-----------	|-------------	           |
-|sender        |[User Object](https://github.com/diem/dip/blob/master/dips/dip-10.md#user-object)  | Y            | Information about the sender |
-|receiver_id   |str          | Y            | Diem ID of the receiver |
-
-**Response:**
-```json
 {
-   "receiver": "<User Object>"
+   "_ObjectType": "CommandRequestObject",
+    "command_type": "ReferenceIDCommand",
+    "command": {
+	    "_ObjectType": "ReferenceIDCommand",
+	    "sender": "alice@avasp",
+	    "sender_address": "f72589b71ff4f8d139674a3f7369c69b",
+	    "recipient": "bob@bvasp",
+	    "reference_id": "5b8403c9-86f5-3fe0-7230-1fe950d030cb", 
+    },
+    "cid": "12ce83f6-6d18-0d6e-08b6-c00fdbbf085a",
 }
 ```
 
-| Field 	   | Type 	     | Required? 	| Description 	             |
-|-------	   |------	     |-----------	|-------------	             |
-|receiver      |[User Object](https://github.com/diem/dip/blob/master/dips/dip-10.md#user-object)  | Y            | Information about the receiver |
-
-* Receiver Diem ID domain must belong to the Receiver VASP. Otherwise, Receiver VASP must return the `400` error code.
-* Sender Diem ID must have a Diem ID domain that belongs to Sender VASP. If this is not the case, the Receiver VASP should return the `400` error code 
-* Receiver VASP may optionally choose how much information to share based on the sender Diem ID. If Receiver VASP decides to not share data, it should return an empty `User Object` with only `diem_id` field set
-* If user is not found, the VASP should return error code `404`  
-
-## Pay endpoint
-
-**TODO: This endpoint is going to be merged into DIP 1**
-
-**Pay endpoint** is used when a user is willing to make a money transfer.
-
-**Request:**
-```json
-{
-   "sender": "<User Object>",
-   "receiver_id": "<Receiver_Diem_ID>",
-   "amount": "<amount>",
-   "currency": "<currency>"
-}
+The format of the success response is:
 ```
-
-| Field 	   | Type 	     | Required? 	| Description 	           |
-|-------	   |------	     |-----------	|-------------	           |
-|sender        |[User Object](https://github.com/diem/dip/blob/master/dips/dip-10.md#user-object)  | Y            | Information about the sender |
-|receiver_id   |str          | Y            | Diem ID of the receiver |
-|amount        |integer      | Y            | Amount of coins to transfer |
-|currency      |str          | Y            | Name of the coin |
-
-**Response:**
-```json
 {
-   "receiver_address": "<Receiver VASP on-chain account address>",
-   "reference_id": "<Payment_Reference_ID>",
-   "receiver": "<User Object>"
+   "_ObjectType": "CommandResponseObject",
+    "status": "success",
+    "result": {
+	    "_ObjectType": "ReferenceIDCommandResponse",
+	    "recipient_address": "c5ab123458df0003415689adbb47326d",
+    },
+    "cid": "12ce83f6-6d18-0d6e-08b6-c00fdbbf085a",
 }
 ```
```

Comment:
We don't have a table to describe each field anymore? I think it worth the effort to give simple description, instead of asking reader to derive them from the example code

Reply:
This is for info and pay endpoint which are not in scope for this DIP

Reply:
I mean the attributes table for ReferenceIDCommand, we need a definition for it and looks like we are defining it in this DIP

Reply:
similarly, we also need an ReferenceIDCommandResponse attributes table.